modules/account-lib/test/unit/coin/avaxc/transferBuilder.ts

```diff
@@ -0,0 +1,194 @@
+import should from 'should';
+import { TransferBuilder as AvaxCTransferBuilder } from '../../../../src/coin/avaxc';
+import { AvaxC, Eth } from '../../../../src';
+
+// import * as testData from '../../../resources/avaxc/avaxc';
+const amount = '20000';
+const toAddress = '0x7325A3F7d4f9E86AE62Cf742426078C3755730d5';
+const keyPair = new AvaxC.KeyPair();
+const key = keyPair.getKeys().prv as string;
+
+describe('ARC20 and AVAXERC20 Tokens', () => {
+  describe('Maintest', () => {
+    it('PNG token transfer should succeed', async () => {
```

Comment:
it seems to be a lot of duplicated code. A plausible approach for this type of test is to use "parametric testing".
See https://medium.com/@nyablk97/parameterized-tests-with-jasmine-ecadb2856980 for a simple explanation

Reply:
Done!